app/Http/Middleware/TrackRecentPages.php

```
@@ -0,0 +1,360 @@
+            'source.all' => 'Sources',
+            'destination.index' => 'Destinations',
+            'storage.index' => 'S3 Storages',
+            'tags.show' => 'Tags',
+            'profile' => 'Profile',
+            'terminal' => 'Terminal',
+            'subscription.show', 'subscription.index' => 'Subscription',
+            default => null,
+        };
+
+        return ['label' => $label, 'sublabel' => null];
+    }
```

PHPMD (2.15.0)

The method deriveLabelAndSublabel() has a Cyclomatic Complexity of 28. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)